doc/user_manual/ProbabilityDistributions.tex

```diff
@@ -903,6 +903,44 @@ \subsubsection{1-Dimensional Discrete Distributions.}
 </Distributions>
 \end{lstlisting}
 
+\paragraph{Uniform Discrete Distribution}
+\label{subsec:UniformDiscrete}
+
+The \textbf{UniformDiscrete} distribution is a discrete distribution which describes a random variable 
+that can have $N$ integer values having equal probability value.
+This distribution allows the user to choose two kinds of sampling strategies: with or without replacement.
+
+\specBlock{a}{Uniform Discrete}
+%
+\attrIntro
+\vspace{-5mm}
+\begin{itemize}
+  \itemsep0em
+  \item \nameDescription
+\end{itemize}
+\vspace{-5mm}
+\subnodeIntro
+\begin{itemize}
+  \item \xmlNode{lowerBound}, \xmlDesc{integer, required parameter}, lower bound.
+  \item \xmlNode{upperBound}, \xmlDesc{integer, required parameter}, upper bound.
+  \item \xmlNode{strategy},   \xmlDesc{string,  required parameter}, type of sampling strategy 
+  (orderedWithReplacement or orderedWithoutReplacement).
```

Comment:
As we have discussed today since this is a 1D distribution, I think there is no meaning to call it `ordered`

Reply:
edited